proto/rill/admin/v1/api.proto

```diff
@@ -2552,6 +2557,21 @@ message PullVirtualRepoResponse {
   string next_page_token = 2;
 }
 
+message GetVirtualFileRequest {
+  // The project ID to get the virtual file for.
+  string project_id = 1;
+  // The environment to get the virtual file for.
+  // It is optional. If the call is made with a deployment access token, it defaults to the environment of the deployment. Otherwise, it defaults to "prod".
+  string environment = 4;
+  // The path of the virtual file to get.
+  string path = 2;
+}
```

Comment:
You'll need to add a `bool superuser_force_access` field here and in `PullVirtualRepoRequest` and consume it in the APIs. See other places for an example. 

Otherwise it will not work for `sudo` users who don't have access to the project.